packages/server/src/automations/triggers.ts

```diff
@@ -1,7 +1,7 @@
 import emitter from "../events/index"
 import { getAutomationParams, isDevAppID } from "../db/utils"
 import { coerce } from "../utilities/rowProcessor"
-import { definitions } from "./triggerInfo"
+import { definitions } from "../../../shared-core/src/automations/triggers"
```

Comment:
this should be access via the package at `@budibase/shared-core` - avoids weird build issues down the line.